Migration from 0.17.1-alpha to 0.18.0-alpha fails

## Expected Behavior
Running migration from `0.16.X-alpha` to `0.18.0-alpha` works.
Running migration from `0.17.3-alpha` to `0.18.0-alpha` fails with an exception.

This seems to be due to the `tombstone` field added to Standard Workspace https://github.com/airbytehq/airbyte/pull/2325/ (in 0.17.1?) which isn't expected while migrating since the expected JSON Schema for that object does not contain any references to that field:
https://github.com/airbytehq/airbyte/blob/master/airbyte-migration/src/main/resources/migrations/migrationV0_14_0/airbyte_config/StandardWorkspace.yaml

This is failing even though, 0.18.0 does not relate to StandardWorkspace objects at all.

## Current Behavior
Upgrade tarball successfully

## Logs

```
2021-03-30 10:45:19 INFO i.a.m.MigrationRunner(parse):78 - {} - args: [--input, /config/3d4e8c96-75c3-48b2-ab5e-69e267acafc4.tar.gz, --output, /config/18.tar.gz]
2021-03-30 10:45:19 INFO i.a.m.MigrationRunner(run):50 - {} - Unpacking tarball
2021-03-30 10:45:19 INFO i.a.m.MigrationRunner(run):67 - {} - Running migrations...
2021-03-30 10:45:19 INFO i.a.m.MigrationRunner(run):68 - {} - MigrateConfig{inputPath=/tmp/airbyte_migrate12711066458087605762/uncompressed, outputPath=/tmp/airbyte_migrate12711066458087605762/output, targetVersion='null'}
2021-03-30 10:45:19 INFO i.a.m.Migrate(run):88 - {} - Starting migrations. Current version: 0.17.3-alpha, Target version: null
2021-03-30 10:45:19 INFO i.a.m.Migrate(run):112 - {} - Migrating from version: 0.17.0-alpha to version 0.18.0-alpha.
2021-03-30 10:45:20 WARN c.n.s.JsonMetaSchema(newValidator):338 - {} - Unknown keyword existingJavaType - you should define your own Meta Schema. If the keyword is irrelevant for validation, just use a NonValidationKeyword
Exception in thread "main" java.lang.IllegalArgumentException: Input data schema does not match declared input schema.
	at io.airbyte.migrate.Migrate.lambda$runMigration$1(Migrate.java:139)
	at java.base/java.util.stream.ReferencePipeline$11$1.accept(ReferencePipeline.java:441)
	at java.base/java.util.Iterator.forEachRemaining(Iterator.java:133)
	at java.base/java.util.Spliterators$IteratorSpliterator.forEachRemaining(Spliterators.java:1801)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474)
	at java.base/java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:150)
	at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:173)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.base/java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:497)
	at io.airbyte.migrate.migrations.MigrationV0_18_0.migrate(MigrationV0_18_0.java:81)
	at io.airbyte.migrate.MigrateWithMetadata.migrate(MigrateWithMetadata.java:52)
	at io.airbyte.migrate.Migrate.runMigration(Migrate.java:149)
	at io.airbyte.migrate.Migrate.run(Migrate.java:113)
	at io.airbyte.migrate.MigrationRunner.run(MigrationRunner.java:70)
	at io.airbyte.migrate.MigrationRunner.main(MigrationRunner.java:108)
Caused by: io.airbyte.validation.json.JsonValidationException: json schema validation failed. 
errors: $.tombstone: is not defined in the schema and the schema does not allow additional properties 
```

## Steps to Reproduce
1. Setup an airbyte instance in 0.17.1 or later
1. export
1. migrate to 0.18.0

## Severity of the bug for you
High 

## Airbyte Version
0.18.1-alpha

--- a/airbyte-migration/src/main/java/io/airbyte/migrate/migrations/MigrationV0_18_0.java
+++ b/airbyte-migration/src/main/java/io/airbyte/migrate/migrations/MigrationV0_18_0.java
@@ -50,6 +50,8 @@ import org.slf4j.LoggerFactory;
  * (used by destination to specify append/overwrite/append_dedup)
  *
  * The primaryKey column is filled if available from the stream if defined by source
+ *
+ * Additionally, this migration updates the JSON Schema for StandardWorkspace with a new field 'tombstone' introduced in 0.17.1
  */
 public class MigrationV0_18_0 extends BaseMigration implements Migration {
 

--- a/airbyte-migration/src/main/java/io/airbyte/migrate/migrations/MigrationV0_18_0.java
+++ b/airbyte-migration/src/main/java/io/airbyte/migrate/migrations/MigrationV0_18_0.java
@@ -51,7 +51,8 @@ import org.slf4j.LoggerFactory;
  *
  * The primaryKey column is filled if available from the stream if defined by source
  *
- * Additionally, this migration updates the JSON Schema for StandardWorkspace with a new field 'tombstone' introduced in 0.17.1
+ * Additionally, this migration updates the JSON Schema for StandardWorkspace with a new field
+ * 'tombstone' introduced in 0.17.1
  */
 public class MigrationV0_18_0 extends BaseMigration implements Migration {
 

--- a/airbyte-migration/src/main/java/io/airbyte/migrate/migrations/MigrationV0_18_0.java
+++ b/airbyte-migration/src/main/java/io/airbyte/migrate/migrations/MigrationV0_18_0.java
@@ -28,11 +28,14 @@ import com.fasterxml.jackson.databind.JsonNode;
 import com.fasterxml.jackson.databind.node.ObjectNode;
 import com.google.common.collect.ImmutableMap;
 import io.airbyte.commons.json.Jsons;
+import io.airbyte.commons.resources.MoreResources;
 import io.airbyte.commons.util.MoreIterators;
 import io.airbyte.migrate.Migration;
 import io.airbyte.migrate.ResourceId;
 import io.airbyte.migrate.ResourceType;
+import java.io.IOException;
 import java.util.Collections;
+import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 import java.util.function.Consumer;
@@ -52,6 +55,7 @@ public class MigrationV0_18_0 extends BaseMigration implements Migration {
 
   private static final Logger LOGGER = LoggerFactory.getLogger(MigrationV0_18_0.class);
 
+  private static final ResourceId STANDARD_WORKSPACE_RESOURCE_ID = ResourceId.fromConstantCase(ResourceType.CONFIG, "STANDARD_WORKSPACE");
   private static final ResourceId STANDARD_SYNC_RESOURCE_ID = ResourceId.fromConstantCase(ResourceType.CONFIG, "STANDARD_SYNC");
 
   private static final String MIGRATION_VERSION = "0.18.0-alpha";
@@ -69,8 +73,20 @@ public class MigrationV0_18_0 extends BaseMigration implements Migration {
   }
 
   @Override
+  public Map<ResourceId, JsonNode> getInputSchema() {
+    final Map<ResourceId, JsonNode> outputSchema = new HashMap<>(previousMigration.getOutputSchema());
+    try {
+      outputSchema.put(STANDARD_WORKSPACE_RESOURCE_ID,
+          Jsons.jsonNode(MoreResources.readResource("migrations/migrationV0_18_0/StandardWorkspace.yaml")));
+    } catch (IOException e) {
+      throw new RuntimeException(e);
+    }
+    return outputSchema;
+  }
+
+  @Override
   public Map<ResourceId, JsonNode> getOutputSchema() {
-    return previousMigration.getOutputSchema();
+    return getInputSchema();
   }
 
   @Override

--- a/airbyte-migration/src/main/resources/migrations/migrationV0_18_0/StandardWorkspace.yaml
+++ b/airbyte-migration/src/main/resources/migrations/migrationV0_18_0/StandardWorkspace.yaml
@@ -1,0 +1,41 @@
+---
+"$schema": http://json-schema.org/draft-07/schema#
+"$id": https://github.com/airbytehq/airbyte/blob/master/airbyte-config/models/src/main/resources/types/StandardWorkspaceConfiguration.yaml
+title: StandardWorkspace
+description: workspace configuration
+type: object
+required:
+  - workspaceId
+  - name
+  - slug
+  - initialSetupComplete
+additionalProperties: false
+properties:
+  workspaceId:
+    type: string
+    format: uuid
+  customerId:
+    type: string
+    format: uuid
+  name:
+    type: string
+  slug:
+    type: string
+  email:
+    type: string
+    format: email
+  initialSetupComplete:
+    type: boolean
+  anonymousDataCollection:
+    type: boolean
+  news:
+    type: boolean
+  securityUpdates:
+    type: boolean
+  displaySetupWizard:
+    type: boolean
+  tombstone:
+    description:
+      if not set or false, the configuration is active. if true, then this
+      configuration is permanently off.
+    type: boolean

